tests/test_main.py

```diff
@@ -101,3 +109,17 @@ def test_rate_service(
         f"'count_of_ratings' should equal to {result_num_ratings}, "
         f"when initial 'count_of_ratings' was {init_num_ratings}"
     )
+
+
+def test_unnecessary_comment():
+    if os.path.exists(os.path.join(os.pardir, "app", "main.py")):
+        main_path = os.path.join(os.pardir, "app", "main.py")
+    else:
+        main_path = os.path.join("app", "main.py")
+
+    with open(main_path, "r") as main:
+        main_content = main.read()
+
+        assert (
+                "# write your code here" not in main_content
+        ), "Remove unnecessary comment"
```

Comment:
This test checks for an unnecessary comment in the `main.py` file. Ensure that the comment `# write your code here` is not present in the file, as this test will fail if it is.